Runtime/Scripts/VideoStreamTrack.cs

```diff
@@ -28,7 +28,7 @@ public class VideoStreamTrack : MediaStreamTrack
         /// <summary>
         /// Flip vertically received video, change it befor start receive video
         /// </summary>
-        public static bool NeedReceivedVideoFlipVertically { get; set; } = false;
+        public static bool NeedReceivedVideoFlipVertically { get; set; } = true;
```

Comment:
I think that we need to set `false` if textures are copied vertically flipped by default.

Reply:
Before #971 it was true received video flip and sender video needFlip is also true (flip vertically and send, upon receipt of flip back). So "true" here is correct.

Reply:
@karasusan 
Ah, I understood.
I am checking this change with Unity Render Streaming, and it is needed to change like this. It should be no problem.
https://github.com/Unity-Technologies/UnityRenderStreaming/pull/953